### PR TITLE
Feature/hide side nav

### PR DESCRIPTION
**Pull request by ahaurw01**
Addresses #27 
I added some breadcrumbs up there since there wasn't a whole lot of context otherwise. It's totally a half-baked idea at the moment, so feel free to shoot down or suggest alternatives. Not sure what looks best there.

### CODE REVIEW

File: app/styles/app.scss [hunk 1]
```diff
@@ -13,6 +13,7 @@
 @import 'components/x-card';
 
 $primary-color-palette: $color-orange;
+$locked-breakpoint: 5000px;
```
Comment (SirZach) [inline app/styles/app.scss]: what does this guy do?

**Comment by SirZach** [comment]
2 thoughts
1) do you think we should auto expand the sidenav once your monitor reaches ridiculous widths? that's actually something I like about it now that it hides and shows depending on width and maybe for our app we just need to increase the width at which it is allowed to show
2) after bringing out the sidenav and navigating, the sidenav stays out with an overlay on top of the rest of the screen instead of automagically hiding.

Secret third thought: I like the breadcrumbs but wonder if we can maybe make that automatic based on the route name? I think routes' have a <code>routeName</code> property that identifies who it is. Also, the text of the bread crumb is very close to the buttons so we either need to more elaborately define the buttons or maybe shrink the breadcrumb text.

File: app/styles/app.scss (hunk 1 above)
Comment (ahaurw01) [inline app/styles/app.scss]: This was the variable ember-paper is using for the min-width on showing the sidenav. I set it really high to effectively force it hidden all the time. We could set this to something more reasonable, like 1900px or something, so people with big monitors aren't inconvenienced.

**Comment by ahaurw01** [comment]
1. Definitely/see comment in styles above.
1.  Yes, that's pretty silly. I'll fix that.
1. 
  1. That's definitely an easier approach than requiring each route to specify its breadcrumb info. One thought is having context-aware breadcrumb names like the deck name or the game matchup. Seems like there are a bunch of approaches out there, but [this guy](https://github.com/chrisfarber/ember-breadcrumbs) has an implementation that allows you to bind arbitrary things up there. Similar to what I was doing, but binds the value instead of only retrieving it on `setupController`.
  1. Yep, looks like a jank with the breadcrumb and button styling as I have it. I'll try out shrinking the text size or possibly giving a little margin between buttons and crumbs. Not sure.
  1. What is your thought on the necessity of breadcrumbs?

File: app/styles/app.scss (hunk 1 above)
Comment (SirZach) [inline app/styles/app.scss]: ah yeah i realized that after i was laying in bed last night. nice find and many thanks to ember-paper (@miguelcobain) for this kind of ease of development

**Comment by SirZach** [comment]
3. iii. What say you to not doing bread crumbs?

**Comment by ahaurw01** [comment]
Yeah, I'm cool dropping the crumbs. Just wanted to try it out to see if it would fit well. Maybe we'll think of something to take up a little space on the toolbar on smaller screens so it's not just the hamburger icon + actions.

**Comment by SirZach** [comment]
I think there's some potential to do some fun stuff up there yet to be thought of.

**Comment by ahaurw01** [comment]
Alrighty, updated to hide the nav until 1700px wide. Also, cached in one of my limited observer tokens and have it closing the side nav when you change routes (if you're < 1700px wide). Crumbs have been dropped.

**Pull request merged**